editor/script_editor_debugger.cpp

```diff
@@ -898,6 +898,10 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 					item.name = strings[2];
 					item.script = strings[0];
 					item.line = strings[1].to_int();
+				} else if (strings.size() == 4) {
```

Comment:
A comment showing what the `== 3` and `== 4` cases refer to could be useful :)

Reply:
Added a comment explaining it

Reply:
Seems like you pushed a version without the comment :)

Reply:
Oh yeah :/ Pushed